articles/140_topic_and_service_name_mapping.md

```diff
@@ -359,10 +359,10 @@ You can read more about partitions in RTI's documentation:
 
 Trade-offs (in comparison to using the whole ROS name along with the namespaces):
 
-- Splitting the ROS name into "namespace" and "base name", and placing the complete namespace into a single partition seemed un-natural.
+- Splitting the ROS name into "namespace" and "base name", and placing the complete namespace into a field designed for another purpose seemed incorrect.
```

Comment:
So I actually still think that we have to justify this statement more precisely. So one concrete reason why putting the namespace in a partition is incorrect is that the tuple of (partition, topic, type) has to be unique amongst a DDS environment. Fast-RTPS does not enforce this, however RTI does, hence the github issue below.